go/vt/mysqlctl/filebackupstorage/file.go

```diff
@@ -58,6 +62,23 @@ type FileBackupHandle struct {
 	errors   concurrency.AllErrorRecorder
 }
 
+func NewBackupHandle(
```

Comment:
remind me why we have this method. Is it so we can assign default with noparam (backward compatibility)?

Reply:
Yeah that's pretty much why. Makes it less likely to introduce nil dereference errors.